src/gui/explorer.go

```diff
@@ -123,10 +125,18 @@ func coinSupply(gateway *daemon.Gateway, w http.ResponseWriter, r *http.Request)
 		return nil, nil
 	}
 
+	totalCoinHours, err := gateway.GetTotalCoinhours()
```

Comment:
Should call GetUnspentOutputs() directly and reuse the result to calculate total coin hours and also filter the unlocked addresses which is done in the lines above

Reply:
when u say filter, do u mean skip the unlocked addresses?